### ISSUE TITLE
Hardwired "bin" as a Robolectric workaround?

### ISSUE THREAD

**Issue by rkawala** [pardom-zz/ActiveAndroid#63]
I'm using Robolectric on my project, and it's very kind of you to have inserted some code that makes it possible to run under Robolectric. I was wondering why "bin" is hardwired into the code, though. Is that an Eclipse thing? IntelliJ IDEA uses "out" by default, it appears.

Thanks,
Rick

**Comment by pardom-zz** [comment]
Yes, that's why. Honestly, I haven't done much testing with Roboelectric, and I haven't run the tests since I recently switched to IntelliJ. I'll look into this more, especially since I sort of hacked together Roboelectric support.

**Comment by pardom-zz** [comment]
Actually, now that I look at it again, I think "bin" is the Roboelectric framework directory for the APK classes. Nevertheless, I'll be looking into this more to make sure it's working regardless of IDE.
